src/main/java/net/obvj/performetrics/util/Duration.java

```diff
@@ -419,7 +422,7 @@ public BigDecimal toTimeUnit(TimeUnit timeUnit)
      * @return the total length of the duration in the specified time unit, with a custom
      *         scale, not null
      */
```

Comment:
Regarding the scale param: now, if a negative amount is passed, the default scale will be applied.